browser/extensions/brave_extension_management.cc

```diff
@@ -69,4 +63,11 @@ void BraveExtensionManagement::OnExtensionUnloaded(
     pref_service_->SetBoolean(kIPFSCompanionEnabled, false);
 }
 
+void BraveExtensionManagement::OnTorDisabledChanged() {
```

Comment:
I think tor executables will be deleted immediately after user disables tor option from settings page.

Reply:
Ah, sorry I didn't see STR. This is expected behavior.
Doesn't this make any issue if deletion happens while tor window is opened?

Reply:
It won't have any issues when tor window is opened, it's already loaded into memory.

Reply:
got it.